_posts/2019-11-18-moveit-grasps.md

```diff
@@ -0,0 +1,47 @@
+---
+author: Rob Coleman
+comments: false
+date: 2019-11-18
+layout: post
+title: PickNik and Rapyuta Team Up to Release MoveIt Grasps
+media_type: image
+media_link: /assets/images/blog_posts/moveit_grasps/moveit_grasps.png
+description: "Bringing a major new feature to MoveIt: a grasping library for manipulation"
```

Comment:
we do not usually quote these. will it know to remove the quotes before displaying?

Reply:
I checked on the live website and it works!